setup.py

```diff
@@ -14,8 +14,10 @@
     "xarray",
 ]
 
-test_requirements = ["pytest", "pytest-subtests", "serialbox"]
-ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@2024.04.00"]
+test_requirements = ["pytest", "pytest-subtests", "serialbox", "coverage"]
+ndsl_requirements = [
+    "ndsl @ git+https://github.com/fmalatino/NDSL.git@fix/computegridoption"
```

Comment:
We should get your ndsl branch into `main` and/or released to omit this special reference.

Reply:
I still have this on the test branch and as a draft, as there is a PR in NDSL ([PR 53](https://github.com/NOAA-GFDL/NDSL/pull/53) ) that is a hotfix that is needed before the next release. Once those are merged I will change the branch to point the new release branch.